extract_bioheart_genos_demux.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# pylint: disable=import-error,missing-module-docstring,wrong-import-position
+
+__author__ = 'annacuomo'
```

Comment:
This should always come after import statements, otherwise pylint (and other linters) will complain